akka-persistence-typed/src/main/scala/akka/persistence/typed/javadsl/PersistentBehavior.scala

```diff
@@ -19,7 +19,15 @@ import scala.util.{ Failure, Success }
 
 /** Java API */
 @ApiMayChange
-abstract class PersistentBehavior[Command, Event, State >: Null](val persistenceId: String) extends DeferredBehavior[Command] {
+abstract class PersistentBehavior[Command, Event, State >: Null] private (val persistenceId: String, supervisorStrategy: Option[BackoffSupervisorStrategy]) extends DeferredBehavior[Command] {
+
+  def this(persistenceId: String) = {
+    this(persistenceId, None)
+  }
+
+  def this(persistenceId: String, backoffSupervisorStrategy: BackoffSupervisorStrategy) = {
```

Comment:
Added to constructor to not imply you can change it every time there is a failure